extensions/2.0/Vendor/EXT_manifold/README.md

```diff
@@ -0,0 +1,171 @@
+
+>NOTE: This is the same definition as used in the 3D Manufacturing Format (3MF) for maximum interoperability.
+
+Manifoldness is critical for reliability of geometric algorithms that operate on solid objects, and so is important in CAD, FEA (Finite Element Analysis), CFD (Computational Fluid Dynamics), and more. However, GPU-friendly formats like glTF with indexed triangles lose manifoldness data because anywhere vertex properties change at an edge (e.g. change of materials, UV swatch, or normals at a crease) the entire vertex must be duplicated and referenced separately. This means the only way to determine linked edges is by merging vertices with identical positions, but this geometric check cannot losslessly recreate the topology, as some separate vertices may happen to coincide in position.
+
+This extension allows authors to denote that the contained mesh data is in fact manifold, and adds a small amount of extra data to losslessly recover this manifold mesh by leveraging sparse accessors. 
+
+## Extending Meshes With Manifold Data
+
+A glTF mesh is denoted as a manifold by adding an `EXT_mesh_manifold` extension object to it and obeying the following restrictions:
+
+- all primitives **MUST** use the `TRIANGLES` topology type;
+- same attributes of different mesh primitives **MUST** reference the same accessor;
+- the `indices` property **MUST** be defined for all primitives and the referenced accessors with index data **MUST** reference the same buffer view.
+
+The `manifoldPrimitive` added in the extension is an alternative `primitive` with additional restrictions:
+- it **MUST** use the `TRIANGLES` topology type;
+- it **MUST** have only the `POSITION` `attribute`, which **MUST** reference the same position `accessor` as the other primitives use; 
+- its `indices` **MUST** reference an `accessor` which **MUST** in turn reference the same `bufferView` as the other primitives' `indices`;
+- its `indices` `accessor` **MAY** include a `sparse` accessor that defines which vertices are to be merged, but every index merged **MUST** have identical position values for the before and after vertices referenced, so that the geometry is unchanged;
+- it **MUST NOT** contain a material;
+- it **MUST NOT** contain morph targets, though the other primitives can.
+
+If the `indices` `accessor` is sparse, then `mergeIndices` and `mergeValues` **MUST** be included and reference `accessors` which are equivalent to the sparse indices and values. This small amount of JSON is duplicated here because sparsity is generally considered an internal detail of compression, but here it also represents the relationship between the separate materials and the unified manifold. 
```

Comment:
To what extent should `mergeIndices` and `mergeValues` be equivalent to internal sparse data?
1. Precisely referring to the same data (buffer views, offsets, and component types) or just resolving to equivalent values no matter where they come from?
2. Presumably, the number of elements must match anyway.

---

> the relationship between the separate materials and the unified manifold

What do "separate materials" mean here?

---

What does it mean for a `manifoldPrimitive` to use a non-sparse indices accessor?